plugin/src/main/java/org/jenkinsci/plugins/structs/describable/UninstantiatedDescribable.java

```diff
@@ -0,0 +1,218 @@
+package org.jenkinsci.plugins.structs.describable;
+
+import hudson.model.Describable;
+import org.jenkinsci.Symbol;
+
+import javax.annotation.Nullable;
+import java.io.Serializable;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.TreeMap;
+
+/**
+ * A reflection-like view of a particular {@link Describable} instance.
+ *
+ * @author Kohsuke Kawaguchi
+ */
+public class UninstantiatedDescribable implements Serializable {
+    private String symbol;
+    private String klass;
+    private final Map<String,?> arguments;
+    private transient DescribableModel model;
```

Comment:
You do not need to have anything restoring this in `readResolve`, so it is not really safely `Serializable` if anyone calls methods like `instantiate` later. Did you actually test serializing it?

```groovy
def x = testAttachments('**/*.png')
semaphore 'restart-here'
junit testDataPublishers: [x]
```